### ISSUE TITLE
Weekly note issues

### ISSUE THREAD

**Issue by AngelusDomini** [liamcain/obsidian-calendar-plugin#135]
### TL;DR
1. "Unable to create new file" error shows when opening an existing weekly note. Creating a new weekly note works, but as long as the note exists, opening it would show the error.
2. Only few weekly notes appear as existing in the Calendar plugin, all of which would be displayed as different week numbers, not corresponding to their actual numbers. The number of notes would depend on the number of months where you have created weekly notes for (so if you have at least one note for Jan, Feb and March, you would have three weekly notes appearing as existing in Calendar)
3. Only those few notes are opened when using the "Open previous/next weekly note" hotkeys, ignoring the other existing weekly notes in between.

---

### Full story:

I get the "Unable to create new file" error when I try to open an existing weekly note file (either when I open it from the Calendar plugin or by using this plugin to open this week's note). These existing weekly notes do not appear in the calendar as existing notes, with two exceptions: my first weekly note—which is week 4—and week 7, both of which show up in the calendar as different week numbers, the note for week 4 is shown as week 1 and the note for week 7 is shown as week 6. Here are some screenshots:

![Screen Shot 2021-02-10 at 6 52 03 PM](https://user-images.githubusercontent.com/59588053/107501935-00c34200-6bd3-11eb-908d-b643b1a74367.png)
![Screen Shot 2021-02-10 at 6 52 11 PM](https://user-images.githubusercontent.com/59588053/107501937-00c34200-6bd3-11eb-89c2-585bab536c5c.png)
![Screen Shot 2021-02-10 at 6 51 46 PM](https://user-images.githubusercontent.com/59588053/107501942-03259c00-6bd3-11eb-8364-36fd4551c650.png)

The error only shows up when the note exists, but when I create a non-existing weekly note, it gets created normally but does not show up as existing, and only when opening it again via the Calendar or this plugin does it show the error.

Furthermore, when I use the hotkeys to open the previous and next weekly note, the plugin skips from Week 4 to Week 7 and vice versa, ignoring the other notes—so only two notes open when using the hotkeys.

From what I've seen, any weekly note I create for a specific month is shown as the first week of that month, regardless of it actually being the month's first week; and I seem to be able to open only that weekly note for that month, so any other existing note I open from that month will show the error. Since we're still at February, only two notes "work," but if I create a weekly note for March, say for week 13, it will show as week 10 from the Calendar which is the first week of the month, and the previous and next weekly note hotkeys will open the notes for week 4, 7 and 13 (if you create the note for week 13).

I made this issue here for the Periodic Notes plugin instead of Calendar because it was only when I installed the former that the weekly notes issues appeared.

Here are my settings:
![image](https://user-images.githubusercontent.com/59588053/107512268-89e17580-6be1-11eb-98ae-43aab4ce2acc.png)

Sorry for the long post 🙏  Hope you look into these issues. I appreciate the work that you do!

**Comment by AngelusDomini** [comment]
Thank you @liamcain! The issues regarding the Calendar plugin have pretty much been resolved: existing weekly notes show up as existing and clicking on these from the calendar actually opens them now.

Unfortunately, the issues with Periodic Notes are still there: hotkey for opening this week's note still shows the `Unable to create new file` error even though the note exists, and the hotkeys for opening the next/previous weekly notes still skip from Week 4 to Week 7.

Any more info I can give you to resolve these issues?

**Comment by liamcain** [comment]
Hi, sorry for the wait! I just pushed the new version of the Periodic Notes plugin which should resolve this.
